src/components/Fragments/index.tsx

```diff
@@ -32,11 +32,7 @@ export default function Fragments({ fragments }) {
       }
     }
 
-    children.push(
-      <Flex key={key} direction="column">
-        {fragment}
-      </Flex>
-    );
+    children.push(<Fragment key={key}>{fragment}</Fragment>);
```

Comment:
Changing this to a fragment so our current selector rules for getting headings will work on headings in fragments.